controllers/configurationpolicy_controller_suite_test.go

```diff
@@ -24,7 +24,7 @@ var samplePolicy = policiesv1alpha1.ConfigurationPolicy{
 	},
 	Spec: policiesv1alpha1.ConfigurationPolicySpec{
 		Severity: "low",
-		NamespaceSelector: policiesv1alpha1.Target{
+		NamespaceSelector: policiesv1alpha1.NamespaceSelector{
```

Comment:
Optional: It'd be nice to rename the import of `policiesv1alpha1` to `policyv1`.